matview.c

```diff
@@ -1526,6 +1587,161 @@ rewrite_query_for_distinct_and_aggregates(Query *query, ParseState *pstate)
 	return query;
 }
 
+/*
+ * rewrite_query_for_exists_subquery
+ *
+ * Rewrite EXISTS sublink in WHERE to LATERAL subquery
+ */
```

Comment:
How about adding some comments here and explaining how the query is converted using a simple SQL query example?

Reply:
What about it?

I wonder it would be helpful if we have an example of an original query including an EXISTS clause and the corresponding rewritten query including a LATERAL subquery.